lib/Doctrine/Migrations/AbstractMigration.php

```diff
@@ -93,24 +95,42 @@ public function skipIf(bool $condition, string $message = '') : void
         }
     }
 
+    /**
+     * @throws MigrationException|DBALException
+     */
     public function preUp(Schema $schema) : void
     {
     }
 
+    /**
+     * @throws MigrationException|DBALException
+     */
     public function postUp(Schema $schema) : void
     {
     }
 
+    /**
+     * @throws MigrationException|DBALException
+     */
     public function preDown(Schema $schema) : void
     {
     }
 
+    /**
+     * @throws MigrationException|DBALException
+     */
     public function postDown(Schema $schema) : void
     {
     }
-
+    
```

Comment:
```suggestion
```

Reply:
I should stop doing PR directly on github without cloning/forking and thinking it's faster :/

Fixed !